client/src/components/Fan.js

```diff
@@ -46,10 +47,12 @@ class Fan extends Component {
     try {
       const web3 = new Web3(provider);
       const networkId = await web3.eth.net.getId();
-      const deployedNetwork = FanProxyContract.networks[networkId];
+      const fanProxyAddress = addresses[networkId]
```

Comment:
This is a lot cleaner than what we had before. Nice!

Reply:
Thanks!